sqltap/sqltap.py

```diff
@@ -38,19 +46,30 @@ class QueryStats(object):
     :attr user_context: The value returned by the user_context_fn set
```

Comment:
need comments explaining the meaning of the new parameters that we've defined: start_time, end_time, params_dict, results